packages/flutter/lib/src/rendering/sliver_fixed_extent_list.dart

```diff
@@ -324,6 +322,74 @@ abstract class RenderSliverFixedExtentBoxAdaptor extends RenderSliverMultiBoxAda
   }
 }
 
+/// Test
+class RenderSliverFixedVariableExtentList extends RenderSliverFixedExtentBoxAdaptor{
```

Comment:
The inheritance chain here is a little strange. The class doesn't use `itemExtent` which is central to `RenderSliverFixedExtentBoxAdaptor`. We should clean that up.